### ISSUE TITLE
integrate Masonite events into core 

### ISSUE THREAD

**Issue by josephmancuso** [MasoniteFramework/core#370]
We should be able to add several events that can be listened to throughout the codebase. Let's make Masonite Events a dependency for core (and throw it in the container via the Events Service Provider) and that's it for this issue. Once that is done we will open a new issue for adding specific events

**Comment by josephmancuso** [comment]
it should be as simple as following the installation steps in the Masonite Events docs in the official docs website

**Comment by rickwest** [comment]
Can I take this @josephmancuso? Would love to start contributing and this seems like a nice way in!

**Comment by josephmancuso** [comment]
sure! Let me know if you have any questions. Either drop them here and @ me or in the slack channel is fine :)

**Comment by clsource** [comment]
I think it would be a good idea that the core contains the files of event system inside the repo and not as a dependency. Since it would be a core functionality it makes sense. Also it seems that the events repo Does not have many stars that would affect such merge
